fluentd-image/v1.7/healthy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh -x
+
+# Liveness probe is aimed to help in situarions where fluentd
+# silently hangs for no apparent reasons until manual restart.
+# The idea of this probe is that if fluentd is not queueing or
+# flushing chunks for 5 minutes, something is not right. If
+# you want to change the fluentd configuration, reducing amount of
+# logs fluentd collects, consider changing the threshold or turning
+# liveness probe off completely.
+# soiurce https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml#L58
+
+BUFFER_PATH=${BUFFER_PATH};
+LIVENESS_THRESHOLD_SECONDS=${LIVENESS_THRESHOLD_SECONDS:-300};
+STUCK_THRESHOLD_SECONDS=${STUCK_THRESHOLD_SECONDS:-900};
+
+if [ ! -e ${BUFFER_PATH} ];
+then
+  exit 1;
+fi;
+touch -d "${STUCK_THRESHOLD_SECONDS} seconds ago" /tmp/marker-stuck;
+if [ -z "$(find ${BUFFER_PATH} -type d -newer /tmp/marker-stuck -print -quit)" ];
+then
+  rm -rf ${BUFFER_PATH};
```

Comment:
I'm not 100% sure we can do it like this. Looks we can easily lose logs in a scenario where an output is down for 15 mins, which is not normal, but can happen. We should at least make this configurable somehow.